packages/components/listbox/src/base/listbox-section-base.tsx

```
@@ -5,7 +5,8 @@ import {DividerProps} from "@nextui-org/divider";
 
 import {ListboxItemProps} from "../listbox-item";
 
-export interface ListboxSectionBaseProps<T extends object = {}> extends SectionProps<"ul", T> {
+export interface ListboxSectionBaseProps<T extends object = {}>
```

Biome

[error] Don't use '{}' as a type.

Prefer explicitly define the object shape. '{}' means "any non-nullable value".

(lint/complexity/noBannedTypes)